hpy/tools/autogen/public_api.h

```diff
@@ -232,9 +237,6 @@ int HPyList_Append(HPyContext ctx, HPy h_list, HPy h_item);
 int HPyDict_Check(HPyContext ctx, HPy h);
 HPy HPyDict_New(HPyContext ctx);
 
-/* pyerrors.h */
```

Comment:
The pyerrors.h functions were accidentally split into two sections, so I combined those sections.